frontend/pluginloader.lua

```diff
@@ -1,10 +1,11 @@
+local InfoMessage = require("ui/widget/infomessage")
+local UIManager = require("ui/uimanager")
 local lfs = require("libs/libkoreader-lfs")
 local logger = require("logger")
+local _ = require("gettext")
```

Comment:
Just because it's a low-level module which didn't previously `require()` much (lfs & logger only), and you don't know at which point it may be imported (well, it's used late, but still), it'd be safer to move the `require()`s for UIManager, InfoMessage and _ in the single callback where they are used. (So that when something require("pluginloader"), it doesn't go yet importing UIManager & the widget stuff.)